controllers/ocimachine_controller.go

```diff
@@ -326,6 +326,10 @@ func (r *OCIMachineReconciler) reconcileNormal(ctx context.Context, logger logr.
 		machineScope.Info("Instance is pending")
 		conditions.MarkFalse(machineScope.OCIMachine, infrastructurev1beta2.InstanceReadyCondition, infrastructurev1beta2.InstanceNotReadyReason, clusterv1.ConditionSeverityInfo, "")
 		return reconcile.Result{RequeueAfter: 10 * time.Second}, nil
+	case core.InstanceLifecycleStateStopping, core.InstanceLifecycleStateStopped:
+		machineScope.SetNotReady()
```

Comment:
Do we want to flag this same state for Terminating, Terminated?

Reply:
good point, done.please review